### ISSUE TITLE
Add wsapi complete_transition

### ISSUE THREAD

**Issue by ozten** [mozilla/persona#2678]
Add a new wsapi `complete_transition` -  indicates that a user has been displayed the "transition to primary" screen.

```
AUTHENTICATED (assertion?) POST request

arguments: "email"

Side affect - will update the ? with ?

returns JSON object { success: boolean }
```

**Comment by ozten** [comment]
From lloyd's [comments on a commit](https://github.com/ozten/browserid/commit/a4ba788e057ceca3ef5003fb4559cc4b1eb1a745#commitcomment-2141598):

I think you should:
1. verify the the currently authenticate user owns the email address in question
2. verify the email address specified is a primary email address
3. update the database!  set email.lastUsedAs to `primary`

Then I think in cert_email we need to update database and set email.lastUsedAs to `secondary`

**Comment by lloyd** [comment]
landed in 781184b5f2b6372c0ce44fff046f1719a78553f9
